packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -853,7 +853,12 @@ class ScrollableBase extends Component {
 				for (let i = 0; i < nodes.length; i++) {
 					const nodeBounds = nodes[i].getBoundingClientRect();
 					if (nodeBounds[first] > viewportBounds[first] && nodeBounds[last] < viewportBounds[last]) {
-						Spotlight.focus(nodes[i]);
+						const node = nodes[i];
+						if (node && node.dataset && 'spotlightContainer' in node.dataset) {
+							Spotlight.focus(`@${node.dataset.spotlightId}`);
```

Comment:
I'm not sure that an app or a component should check if a node is a DOM element or a Spotlight container. If can, it would be better to fix Spotlight side. But I don't have much information about Spotlight. So I hope that someone else could check it.

If a node is a Spotlight container and Spotlight.focus(node) is called, then node.focus() in https://github.com/enactjs/enact/blob/develop/packages/spotlight/src/spotlight.js#L231 is called instead of the child node of the node.

Reply:
I've had a similar thought. Officially, `Spotlight.focus()` doesn't support passing a node but it's an oft used method within the framework.

I think I'd be in favor of adding a branch within `Spotlight.focus()` to resolve the container ID when the node is passed.

Should be something like the following added to `Spotlight.focus()`:
 ```
			} else if (isContainer(elem)) {
				target = getTargetByContainer(getContainerId(elem));
			}
```